contracts/strategies/dnsVectorStrategy/DnsVectorStrategyAumLib.sol

```diff
@@ -46,19 +48,15 @@ library DnsVectorStrategyAumLib {
             strategyStorage.aAaveSupplyToken.balanceOf(address(this))
         );
 
-        // get Traker Joe LP token reserves
-        // asuming a certain order of tokens
+        // get Pangolin LP token reserves
+        // assuming a certain order of tokens
         (
             uint112 reserve0, // wAvax (generally it can be aaveSupplyToken or aaveBorrowToken)
             uint112 reserve1, // usdc (generally it can be aaveSupplyToken or aaveBorrowToken)
 
-        ) = strategyStorage.traderJoePair.getReserves();
-        uint256 lpTokenTotalSupply = strategyStorage
-            .traderJoePair
-            .totalSupply();
-        uint256 lpTokenContractBalance = strategyStorage
-            .vectorPoolHelperJoe
-            .balanceOf(address(this));
+        ) = strategyStorage.pangolinPair.getReserves();
+        uint256 lpTokenTotalSupply = strategyStorage.pangolinPair.totalSupply();
```

Comment:
pangolin LP tokens have the same 'decimal' number as TraderJoe, right?

Reply:
Yes.